prov/gni/src/gnix_rma.c

```diff
@@ -881,6 +1082,8 @@ ssize_t _gnix_rma(struct gnix_fid_ep *ep, enum gnix_fab_req_type fr_type,
 	int rc;
 	int rdma;
 	struct fid_mr *auto_mr = NULL;
+	struct gnix_fab_req *more_req;
+	struct slist_entry *sle;
 
```

Comment:
That makes more sense.  I was just thinking it might prevent people from tripping over it, but there's really nothing to trip.